sql/setup.sql

```diff
@@ -1,2 +1,24 @@
 -- Use this file to define your SQL tables
 -- The SQL in this file will be executed when you run `npm run setup-db`
+
+DROP table if EXISTS users;
+DROP table if EXISTS secrets;
+
+CREATE table users(
+  id BIGINT GENERATED ALWAYS as IDENTITY PRIMARY KEY,
+  first_name VARCHAR NOT NULL,
+  last_name VARCHAR NOT NULL,
+  email VARCHAR NOT NULL,
+  password_hash VARCHAR NOT NULL
+);
+
+CREATE table secrets(
+  id BIGINT GENERATED ALWAYS as IDENTITY PRIMARY KEY,
+  title VARCHAR NOT NULL,
+  description VARCHAR NOT NULL,
+  created_at TIMESTAMP
+);
```

Comment:
It appears the timestamp function is not generating a timestamp, let's try using this CURRENT_TIMESTAMP() instead of TIMESTAMP.